packages/web-components/fast-foundation/package.json

```diff
@@ -27,6 +27,10 @@
       "default": "./dist/esm/index.js"
     },
     "./custom-elements.json": "./dist/custom-elements.json",
+    "./design-token-core": {
```

Comment:
I want to keep them separate. To start, there are some exports with the same name so that would make conflicts or force us to change the name. Additionally, I think *most* folks don't need to use the core infrastructure, so keeping that separate from the primary package makes sense to me.